packages/http/src/router/__tests__/matchBaseUrl.spec.ts

```diff
@@ -136,7 +137,7 @@ describe('matchServer.ts', () => {
   describe('convertTemplateToRegExp()', () => {
     test('given no variables should resolve to the original string', () => {
       const regexp = convertTemplateToRegExp('{a}');
-      expect(regexp).toEqual(/^{a}$/);
+      expect(regexp).toEqual(E.right(/^{a}$/));
```

Comment:
We have a dedicated helper for this — check out the `assertRight` function

Reply:
Oh, that's what I was looking for... Thank you, I'll clean those up.